File: Java/Day02Assignment/Tshirt.java
package com.sonata;

public class Tshirt {
	String color;
	String material;
	String design;
	public void display() {
		System.out.println(color);
		System.out.println(material);
		System.out.println(design);
	}
	Tshirt (String s, String r,String t){
		this.color=s;
		this.material=r;
		this.design=t;
	}
	Tshirt(){}
	public static void main(String args[]) {
		Tshirt t1=new Tshirt();
		t1.color="red";
		t1.material="cotton";
		t1.design="halfsleeves";
		t1.display();
		Tshirt t2=new Tshirt("blue","nylon","fullsleeves");
		t2.display();
		Tshirt t3=new Tshirt("yellow","cotton","halfsleeves");
		t3.display();
		
		
	}

}

File: TypeScript/TShirt.ts
class TShirt{
    color:string;
    material:string;
    design:string;
    size:string;
    constructor(c:string,m:string,d:string,s:string){
        this.color=c;
        this.material=m;
        this.design=d;
        this.size=s;
    }
    display(){
        console.log("Color="+this.color);
        console.log("Design="+this.design);
        console.log("Material="+this.material);
        console.log("Size"+this.size);
    }
}
var t1=new TShirt("blue","cotton","plain","small");
t1.display();
var t2= new TShirt("red","woolen","checks","large");
t2.display();
var t3=new TShirt("green","nylon","box","Xtra-Large");
t3.display();

